rotkehlchen/exchanges/coinbase.py

```diff
@@ -538,7 +538,7 @@ def _query_single_account_transactions(
             elif tx_type in ('buy', 'sell'):
                 self._process_normal_trade(event=transaction, trades=trades)
             elif (
-                    tx_type in ('interest', 'inflation_reward') or
```

Comment:
If this exists as a transaction type then it's a good addition.

Can you also edit the test at `test_coinbase_query_income_loss_expense` and add a mock event with this type and test it in the test?

Reply:
Will do!

Reply:
Done. This is how I get the data from the API, unfortunately without any notes